examples/nlp/intent_detection_slot_tagging/data/assistant_utils.py

```diff
@@ -0,0 +1,157 @@
+# =============================================================================
+# Copyright 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+import os
+import re
+import shutil
+
+from nemo import logging
+from nemo.collections.nlp.data.datasets.datasets_utils.data_preprocessing import (
+    DATABASE_EXISTS_TMP,
+    if_exist,
+    write_files,
+)
+
+
+# put training files in convenient place for conversion to our format
+def copy_input_files(infold):
+    our_infold = infold + "/dataset"
```

Comment:
could you please move comments inside docstings

Reply:
Corrected